src/main/java/net/fabricmc/loom/AbstractPlugin.java

```diff
@@ -58,344 +67,346 @@
 import java.util.Set;
 import java.util.function.Predicate;
 
+import groovy.util.Node;
+
 public class AbstractPlugin implements Plugin<Project> {
-	protected Project project;
+    protected Project project;
+
+    public static boolean isRootProject(Project project) {
+        return project.getRootProject() == project;
+    }
 
-	private void extendsFrom(String a, String b) {
-		project.getConfigurations().getByName(a).extendsFrom(project.getConfigurations().getByName(b));
-	}
+    private void extendsFrom(String a, String b) {
```

Comment:
retain tabs please!

Reply:
Discussion on the Discord about this here: https://discordapp.com/channels/507304429255393322/566418023372816394/587800276791197716

Reply:
The inconsistency in Loom is straying from all tabs to rogue spaces, not the other way around

Reply:
Then what's the point of the `.editorconfig`? I thought it would be standardized there if it was a thing

Reply:
@asiekierka has indicated that fabric projects will stick to tabs, from java files, yarn buildscript, and even enigma and tiny mappings.

Reply:
There are no formal formatting declarations anywhere besides a few ancient (in modern Fabric terms) ones which have never really used. On the whole things use tabs, but there are exceptions like Stitch.

Whilst it doesn't cover `.java` files, the [Loom `.editorconfig`](https://github.com/FabricMC/fabric-loom/blob/dev/0.2.4/.editorconfig) does only state tabs too for what it's worth.